### PR TITLE
Update en_GB.ts

### PR DESCRIPTION
**Pull request by Gooochy**
Changed Stuff I missed/capitalised certain things

### CODE REVIEW

**Comment by JRH-1997** [comment]
@Gooochy schoolings at the vehicles need to be exact the names ingame. That's why there were names not capitalized

**Comment by Gooochy** [comment]
> @Gooochy schoolings at the vehicles need to be exact the names ingame. That's why there were names not capitalized

Okay will the rest be added?

File: src/i18n/en_GB.ts [hunk 1]
```diff
@@ -202,14 +202,14 @@ export default {
...
-            caption: 'Police helicopter',
+            caption: 'Police Helicopter',
             color: '#0a7c16',
             coins: 30,
             credits: 300_000,
             minPersonnel: 1,
             maxPersonnel: 3,
-            schooling: 'Police - Police aviation',
-            shownSchooling: 'Police aviation',
+            schooling: 'Police - Police Aviation',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:211]: back to old

File: src/i18n/en_GB.ts [hunk 2]
```diff
@@ -202,14 +202,14 @@ export default {
...
+            shownSchooling: 'Police Aviation',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:212]: back to old

File: src/i18n/en_GB.ts [hunk 3]
```diff
@@ -218,8 +218,8 @@ export default {
             credits: 7_000,
             minPersonnel: 1,
             maxPersonnel: 2,
-            schooling: 'Police - Dog handling',
-            shownSchooling: 'Dog handling',
+            schooling: 'Police - Dog Handling',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:221]: back to old

File: src/i18n/en_GB.ts [hunk 4]
```diff
@@ -218,8 +218,8 @@ export default {
...
+            shownSchooling: 'Dog Handling',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:222]: back to old

File: src/i18n/en_GB.ts [hunk 5]
```diff
@@ -229,8 +229,8 @@ export default {
             credits: 7_000,
             minPersonnel: 1,
             maxPersonnel: 4,
-            schooling: 'Police - Firearms training',
-            shownSchooling: 'Firearms training',
+            schooling: 'Police - Firearms Training',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:232]: back to old

File: src/i18n/en_GB.ts [hunk 6]
```diff
@@ -229,8 +229,8 @@ export default {
...
+            shownSchooling: 'Firearms Training',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:233]: back to old

File: src/i18n/en_GB.ts [hunk 7]
```diff
@@ -249,7 +249,7 @@ export default {
             credits: 25_500,
             minPersonnel: 1,
             maxPersonnel: 6,
-            schooling: 'Fire Station - Mobile command',
+            schooling: 'Fire Station - Mobile Command',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:252]: back to old

File: src/i18n/en_GB.ts [hunk 8]
```diff
@@ -307,9 +307,9 @@ export default {
             credits: 4_000,
             minPersonnel: 1,
             maxPersonnel: 1,
-            schooling: 'Rescue - Critical care',
-            shownSchooling: 'Critical care',
-            special: 'Can only be placed at the home response location',
+            schooling: 'Rescue - Critical Care',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:310]: back to old

File: src/i18n/en_GB.ts [hunk 9]
```diff
@@ -307,9 +307,9 @@ export default {
...
+            shownSchooling: 'Critical Care',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:311]: back to old

File: src/i18n/en_GB.ts [hunk 10]
```diff
@@ -307,9 +307,9 @@ export default {
...
+            special: 'Can only be placed at the Home Response Location',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:312]: It can stay that way. Thank you

File: src/i18n/en_GB.ts [hunk 11]
```diff
@@ -318,7 +318,7 @@ export default {
             credits: 2_500,
             minPersonnel: 1,
             maxPersonnel: 1,
-            special: 'Can only be placed at the home response location',
+            special: 'Can only be placed at the Home Response Location',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:321]: It can stay that way. Thank you

File: src/i18n/en_GB.ts [hunk 12]
```diff
@@ -345,8 +345,9 @@ export default {
             credits: 19_000,
             minPersonnel: 1,
             maxPersonnel: 2,
-            schooling: 'Police - Firearms training',
-            shownSchooling: 'Firearms training',
+            schooling: 'Police - Firearms Training',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:348]: back to old

File: src/i18n/en_GB.ts [hunk 13]
```diff
@@ -345,8 +345,9 @@ export default {
...
+            shownSchooling: 'Firearms
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:349]: back to old at one line

File: src/i18n/en_GB.ts [hunk 14]
```diff
@@ -875,7 +876,7 @@ export default {
             vehicles: {
                 'Ambulances': [5],
                 'HEMS': [9],
-                'First Responder': [10, 19, 21, 22],
+                'Rapid Response Vehicles': [10, 19, 21, 22],
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:879]: It can stay that way. Thank you

File: src/i18n/en_GB.ts [hunk 15]
```diff
@@ -945,15 +946,15 @@ export default {
         ],
         'Police': [
             {
-                caption: 'Police aviation',
+                caption: 'Police Aviation',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:949]: back to old

File: src/i18n/en_GB.ts [hunk 16]
```diff
@@ -945,15 +946,15 @@ export default {
...
                 duration: '7 Days',
             },
             {
-                caption: 'Firearms training',
+                caption: 'Firearms Training',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:953]: back to old

File: src/i18n/en_GB.ts [hunk 17]
```diff
@@ -945,15 +946,15 @@ export default {
...
                 duration: '5 Days',
             },
             {
-                caption: 'Dog handling',
+                caption: 'Dog Handling',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:957]: back to old

File: src/i18n/en_GB.ts [hunk 18]
```diff
@@ -963,7 +964,7 @@ export default {
         ],
         'Rescue': [
             {
-                caption: 'Critical care',
+                caption: 'Critical Care',
```
Comment (Suchty112) [inline src/i18n/en_GB.ts:967]: back to old

**Review by Suchty112 — CHANGES_REQUESTED** [review]

**Review by Suchty112 — APPROVED** [review]

**Pull request merged**